src/oss/confluent.sh

```diff
@@ -911,7 +911,26 @@ list_command() {
     fi
 }
 
+validate_java_version() {
+    local target_service=${1}
+    if [[ $target_service = "zookeeper" || target_service = "kafka" ]]; then
+        return
+    fi
+    local java_version=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;')
+    # Warn users if it is not Java7 or Java8, or if we can't figure it out.
+    if [[ $java_version -ne 17 && $java_version -ne 18 ]]; then
+        echo "WARNING: Java version 1.8 is recommended:"
+        echo "  See https://docs.confluent.io/current/installation/versions-interoperability.html for detailed version and interoperability information"
+    fi
+    # Fail fast if the version is definitely Java9
+    if [[ $java_version = 19 ]]; then
```

Comment:
java `10.0.1` was released last week. we might want to add that case here too.

Reply:
Also, when running the sed command with Java 10 on my computer I get this:

```
$ /Library/Java/JavaVirtualMachines/jdk-10.0.1.jdk/Contents/Home/bin/java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;'
100 2018-04-17
```

I think we should simplify the sed command. It will be easier to read/modify if the processing is done in multiple steps. A suggestion: 

```
java -version 2>&1  | head -n 1 | awk -F '"' '{print $2}' | awk -F'.' '{OFS = ""; print $1,$2}'
```

Reply:
Will do